bin/upgrade/DEV/scripts/convert_member_emails_to_lowercase.js

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
Add also path to this here:
https://github.com/Countly/countly-server/blob/next/bin/upgrade/DEV/upgrade_db.sh#L39